src/cloudimagedirectory/transform/transform.py

```diff
@@ -14,10 +14,6 @@
 class Pipeline:
     """Builds a pipeline of transformer tasks."""
 
-    transformers: list[Callable] = []
-    filter_funcs: list[Callable] = []
-    idx_generators: list[Callable] = []
```

Comment:
Have you checked other classes to make sure that they don't define class attributes as well?